dev-docs/get-started-with-prebid-server.md

```diff
@@ -121,13 +121,13 @@ pbjs.que.push(function() {
 {% endhighlight %}
 
 {: .alert.alert-info :}
-**OpenRTB Endpoint**  
-If your `s2sConfig.endpoint` points to a url containing the path `openrtb2/auction`, such as the AppNexus-hosted endpoint https://prebid.adnxs.com/pbs/v1/openrtb2/auction', Prebid will communicate with that endpoint using the OpenRTB protocol.  
+**OpenRTB Endpoint**
+If your `s2sConfig.endpoint` points to a url containing the path `openrtb2/auction`, such as the AppNexus-hosted endpoint https://prebid.adnxs.com/pbs/v1/openrtb2/auction', Prebid will communicate with that endpoint using the OpenRTB protocol.
```

Comment:
it's just `/openrtb2/` now. https://github.com/prebid/Prebid.js/blob/master/modules/prebidServerBidAdapter.js#L658

Reply:
Fixed

Reply:
Not https://github.com/prebid/Prebid.js/blob/master/modules/prebidServerBidAdapter.js#L41 ?

Reply:
Both are right :). The PBS endpoint is `/openrtb2/auction`, but `prebid.js` has logic to use OpenRTB `POST` data if `/openrtb2/` exists in the path.

It's _very_ unlikely to have client-facing impact, since `/openrtb2/auction` is the only endpoint in PBS which includes `/openrtb2/` in the path... but this does more correctly describe the `prebid.js` logic.